### PR TITLE
Rachel

### PR DESCRIPTION
**Pull request by rachellindsay**
Added pick_6 version 1 and weather_api version 1

### CODE REVIEW

File: code/rachel/python/pick_6/pick_6_1.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    # runs the winner()function that calculates whether there are any winnings with each loop.
+    winnings = winner(matches) 
+    # runs the profit_or_loss() function to determine the running total of funds won or lost as each loop completes, subtracting 2 for each loop and adding any winnings.
+    balance = profit_or_loss(balance, winnings)
+    # use x to increment the loop
+    x += 1
+else:
+    False
+
+print(f"Your final balance is {balance}.")
```
Comment (AlexDeMichieli) [inline code/rachel/python/pick_6/pick_6_1.py:95]: Missing v2

**Review by AlexDeMichieli — CHANGES_REQUESTED** [review]
Requesting Pick 6 v2 before merge.

**Comment by rachellindsay** [comment]
I added Pick 6 version 2.

**Review by ttajohnson — APPROVED** [review]
Merging for Pick 6 completion.

**Pull request merged**